### DISCUSSION TITLE
Getting Confused with [] [[ ]] while using loc()

### DISCUSSION THREAD

**Discussion by AzamBukhari** [AammarTufail/python_ka_chilla2023#54] [Q&A]
![image](https://user-images.githubusercontent.com/121688289/211844395-fdb22502-b07e-49b3-8e05-e63c4fea5938.png)
In both cases multiple arguments are being passed, but in scenario one we are using double [[]] and scenario two []; why
How to deal with this confusion

# now hot to select rows
df.describe().loc[['min', '25%', '50%', '75%', 'max']]

# now hot to select rows method number 2
df.describe().loc['min': 'max']

**Comment by Muhammad-Ali-Butt** [comment]
> df.describe().loc[['min', '25%', '50%', '75%', 'max']]
> df.describe().loc['min': 'max']
> In both cases multiple arguments are being passed, but in scenario one we are using double [[]] and scenario two []; why
How to deal with this confusion


No bro , in the second case its only one being passed which is max : min(max to min). btw in the first case the argument being passed is a List of index labels or index names and using [[]] returns the dataframe. you can also pass a single label for row and column for example if u have a column named A then df.describe().loc['50%', 'A'] and this will return the specifc value in the row 50% and column A.
if multiple were being passed in case 2 then it would look something like this: for example if df.descibe had columns col_A and col_B **df.describe().loc['min': 'max', ['col_A' : 'col_B' ]]**

**Reply by usamabajwa86** [reply]
in first command your are passing multiple arguments however in 2nd you are giving range from min to max.

all other arguments are between min and max range

**Reply by Muhammad-Ali-Butt** [reply]
> in first command your are passing multiple arguments however in 2nd you are giving range from min to max.
> 
> all other arguments are between min and max range

@usamabajwa86 bhai first command mai argument aik hi ha jo basically aik list ha index names/labels ki aur [[]] se uska dataframe display ho jata ha aur jo 2nd range ha min to max wali wo bhi single argument ha jis mai min to max jo khud 2 index labels/names hain unki range mai vals display hoti ha. 
mujhay bhi 1st walay mai galati lagi thi but dobara dekha
